### ISSUE TITLE
Implement first version of Toggle Group (Segmented Control)

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#305]
### Acceptance criteria

- It should be implemented in accordance [with the designs in Figma](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=11508-16091&p=f&t=BFJvHaOjjowjKGIw-11)
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs (e.g form interactions)
- It should have appropriate component level design tokens that references semantic, and if appropriate, primitive level tokens.

<img width="1344" height="734" alt="Image" src="https://github.com/user-attachments/assets/9499009e-fe6b-4e44-8950-70303a16866b" />

### Additional information

- Some inspiration/checklists in regards of Accessibility:
   - [Radio Group Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/radio/) and [Toolbar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/)
   - [Primer: Segmented Control Accessibility](https://primer.style/product/components/segmented-control/accessibility/)
   - [Sanoma: Toggle Group Accessibility](https://sanomalearning.design/categories/components/toggle-group/accessibility/)
   - [Entur: Segmented Control Tilgjengelighet](https://linje.entur.no/komponenter/skjemaelementer/segmentedcontrol)
   - [Roving tabindex technique](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Keyboard-navigable_JavaScript_widgets#technique_1_roving_tabindex)

**Comment by dafn** [comment]
Denne er etterspurt av hedda
